src/libraries/System.Configuration.ConfigurationManager/tests/System/Configuration/CustomHostTests.cs

```diff
@@ -31,14 +31,12 @@ public void FilePathIsPopulatedCorrectly()
         }
 
         /// <summary>
-        /// Makes Assembly.GetEntryAssembly() return null using private reflection.
+        /// Makes Assembly.GetEntryAssembly() return null by passing the null literal
+        /// to Assembly.SetEntryAssembly().
         /// </summary>
         private static void MakeAssemblyGetEntryAssemblyReturnNull()
```

Comment:
I think you can delete this method and just call `Assembly.SetEntryAssembly(null);` in the one place that it is used. (The factoring into separate method made sense when the method was more complicated.)